streams/id.m3u

```diff
@@ -10,11 +10,17 @@ https://alfa.kugo.id/playout/ainos.stream/chunklist.m3u8
 https://tv.rodja.live/aliman/ngrp:HD_all/playlist.m3u8
 #EXTINF:-1 tvg-id="ANTV.id",ANTV (480p) [Geo-blocked]
 http://210.210.155.37/qwr9ew/s/s07/index1.m3u8
+#EXTINF:-1 tvg-id="ANTV.id",ANTV (1080p) [Geo-blocked]
+https://tv.yankie.my.id/cdn/antv.m3u8
 #EXTINF:-1 tvg-id="AshiilTV.id" user-agent="Mozilla/5.0 (X11; Linux x86_64)",Ashiil TV (360p) [Geo-blocked]
 #EXTVLCOPT:http-user-agent=Mozilla/5.0 (X11; Linux x86_64)
 https://h1.intechmedia.net/intech/ch2.m3u8
 #EXTINF:-1 tvg-id="BaliTV.id",Bali TV (720p) [Geo-blocked]
 https://cdn-telkomsel-01.akamaized.net/Content/HLS/Live/channel(8e867ae0-b2c0-4968-9f60-a11aee8c0987)/index.m3u8
+#EXTINF:-1 tvg-id="BandungTV.id",Bandung TV
+http://202.150.153.254:65500/bandungtvWEBSITE.m3u8
+#EXTINF:-1 tvg-id="BantenTV.id",Banten TV (720p)
+https://5bf7b725107e5.streamlock.net/bantentv/bantentv/playlist.m3u8
```

Comment:
Offline.

<img width="502" alt="image" src="https://github.com/iptv-org/iptv/assets/7253922/bc269ff7-5b6d-4506-848d-ffa5a5b31bcb">

Reply:
https://bantentv.com/live/ seems to be from an official source, maybe not 24/7.

Reply:
@BellezaEmporium @freearhey yeah, it's not 24/7